docs/faq/network/blockchain-architecture.md

```diff
@@ -0,0 +1,103 @@
+
+### What is Proof of Stake?
+
+PoS is a consensus mechanism many blockchain networks use to **validate and add new transactions**. Unlike Proof of Work, which requires miners to solve complex mathematical problems to add new blocks to the blockchain, PoS relies on the amount of cryptocurrency a person holds and is willing to stake as collateral.
+
+In a PoS blockchain, validators are chosen to create a new block based on their **stake in the network**. The more cryptocurrency a person holds, the chances of being selected as a validator increase. Once chosen, validators validate transactions and add them to the blockchain. For their service, validators are rewarded with transaction fees and potentially new coins.
+
+One of the main advantages of PoS over PoW is its **energy efficiency**. While PoW requires massive computational resources and energy consumption, PoS achieves consensus with minimal energy use. The efficiency makes PoS more sustainable and environmentally friendly for blockchain networks.
+
+Another advantage of PoS is the security it provides. Since **validators have a significant investment** in the networks coin, they are incentivized to act honestly. If a validator tries to manipulate the system or validate fraudulent transactions, they risk losing their stake, making attacks on the network costly and, therefore, less likely.
+
+### Why did LUKSO choose Proof of Stake?
+
+LUKSO wanted to start as a **completely decentralized chain without delegating votes**. Proof of Stake makes it easy to allow everyone to participate, even without special hardware. It's the latest and most energy-efficient consensus mechanism for EVM blockchains and is seen as state-of-the-art. It aligns very well with the goals we set for the project.
+
+### Are transactions cheaper than on other blockchains?
+
+Despite the type of network, transactions are usually cheaper on new blockchain networks due to the **reduced competition for block space**. Essentially, every block on a blockchain has a finite capacity for transactions. If the demand for transactions surpasses this capacity, users must compete for the limited space. This competition often manifests as transaction fees. On established networks like Ethereum, transaction fees can be pretty steep.
+
+On newer networks, like LUKSO, fewer users equate to **less competition and lower transaction fees**. Nevertheless, transaction fees are likely to increase as a network attracts more users and demand for block space rises over time.
+
+### What EVM Protocol Upgrades are live on LUKSO?
+
+The LUKSO Mainnet is up to date with [every Ethereum fork](https://ethereum.org/en/history/) until Shanghai, which was released in 2023. Except for Shapella (Capella and Shanghai), all upgrades were integrated within the mainnet launch.
```

Comment:
this paragraph is not up to date I believe, as we did the shapella fork few days ago.

Reply:
I think Johann is right, the paragraph says Shapella is not activated, but it was a few days ago:
https://github.com/lukso-network/network-configs/blob/main/mainnet/shared/genesis.json#L18

<img width="725" alt="image" src="https://github.com/lukso-network/docs/assets/477945/a44bd6e5-9dfd-4224-b640-4125f0783406">

Also @fhildeb for things like that, let's let the person who made the comment write the conversation as resolved.

Reply:
@Hugoo Alright. But the sentence says that all forks are activated... as Shanghai is the last fork included in Shapella. Where does it state Shapella is not activated? The answer states that Shapella was not activated from genesis on, but later. Maybe I should add the timestamp to clear things up.

Reply:
Ok i see, i think if both Johann and I got confused, maybe let's just make it more obvious.

Shapella was activated on XXX.

I read it fast and i read: "All were activated EXCEPT shapella".

Let's do maybe:

```suggestion
The LUKSO Mainnet is up to date with [every Ethereum fork](https://ethereum.org/en/history/) until Shanghai, which was released in 2023. All upgrades were integrated within the mainnet launch, except for Shapella (Capella and Shanghai) which was activated on the 11th of July.
```

Reply:
Great. I just rephrased the sentence a bit more to also include the correct block times. (so there is no confusion when we add future forks)